airc/README.md

```diff
@@ -0,0 +1,55 @@
+# AirScript Compiler
+
+This crate aggregates all components of the AirScript compiler into a single place. Specifically, it re-exports functionality from the [parser](../parser/), [ir](../ir/), and [codegen-winter](../codegen-winter/) crates. Additionally, when compiled as an executable, this crate can be used via a [CLI](#command-line-interface-cli) to transpile AIRs defined in AirScript to a specified target language.
+
+## Basic Usage
+
+An in-depth description of AirScript is available in the full AirScript [documentation](https://0xpolygonmiden.github.io/air-script/).
+
+The compiler has three stages, which can be imported and used independently or together.
+
+1. [Parser](../parser/): scans and parses AirScript files and builds an AST
+2. [`AirIR`](../ir/): produces an intermediate representation from an AirScript AST
+3. Code generation: translate an `AirIR` into a specific target language
+   - [Winterfell Code Generator](../codegen-winter/): generates Rust code targeting the [Winterfell prover](https://github.com/novifinancial/winterfell).
+
+Example usage:
+
+```Rust
+use airc::{parse, AirIR, CodeGenerator};
+
+// parse the source string to a Result containing the AST or an Error
+let ast = parse(source.as_str()).expect("Parsing failed");
+
+// process the AST to get a Result containing the AirIR or an Error
+let ir = AirIR::from_source(&ast).expect("AIR is invalid");
+
+// generate Rust code targeting the Winterfell prover
+let rust_code = CodeGenerator::new(&ir);
+```
+
+To run the full transpilation pipeline, the CLI can be used for convenience
+
+## Command-Line Interface (CLI)
+
+There is a command-line interface available for transpiling AirScript files to Rust. There are also several example `.air` files written in AirScript which can be found in the `examples/` directory.
```

Comment:
Minor nit: It might be better to have the line about example `.air` files somewhere else. I know it's relevant to the docs just below about example usage using CLI but maybe this can be moved to the `Basic Usage` section above below the `Example usage` subsection.

Reply:
I moved it just below the `Example usage`, which felt like it flowed a bit better